scipy/optimize/_basinhopping.py

```diff
@@ -437,6 +437,19 @@ def basinhopping(func, x0, niter=100, T=1.0, stepsize=0.5,
         `take_step` and `accept_test`, and these functions use random
         number generation, then those functions are responsible for the state
         of their random number generator.
+    accept_rate : float, optional
```

Comment:
What is the sensible value range for accept_rate?

Reply:
\> 0 and < 1.  It would probably make sense to raise an exception otherwise.